### PR TITLE
Fixed infer.py for multiple class prediction

### PR DESCRIPTION
**Pull request by danielfennelly**

### CODE REVIEW

File: deepform/model.py [hunk 1]
```diff
@@ -128,7 +129,9 @@ def latest_model():
 
 
 def load_model(model_file=None):
-    filepath = model_file or latest_model()
```
Comment (moredatarequired) [inline deepform/model.py:131]: Did you want to prevent it from loading the latest model by default?

File: deepform/data/tokenize_pdfs.py [hunk 2]
```diff
@@ -110,12 +110,12 @@ def extract_doc(pdf_path, window_len):
     df["tok_id"] = np.minimum(511, df["tok_id"])
     return Document(
         slug=pdf_path.stem,
-        tokens=df[TOKEN_COLS[:-1]],
+        tokens=df,
```
Comment (moredatarequired) [inline deepform/data/tokenize_pdfs.py:113]: This adds some redundant columns, but it's not a big deal.

**Review by moredatarequired — APPROVED** [review]
A couple comments, but LGTM

**Comment by danielfennelly** [comment]
@moredatarequired still troubleshooting some errors that come up when invoking from the command line. Tensorflow or Keras are doing some magic which is changing types out from underneath the code in ways that break things. Working on it before merging.

**Pull request merged**